docs/reference/commandline/create.md

```diff
@@ -69,6 +69,7 @@ Creates a new container.
       -P, --publish-all=false       Publish all exposed ports to random ports
       -p, --publish=[]              Publish a container's port(s) to the host
       --pid=""                      PID namespace to use
+      --pids-limit=0                Tune container pids limit (set -1 for unlimited)
```

Comment:
`-1` means "explicitly set the limit to be unlimited, and error out if the cgroup doesn't exist"
`0` means "use the default" which results in it being unlimited.